control-plane/subcommand/server-acl-init/rules_test.go

```diff
@@ -917,7 +917,31 @@ func TestInjectRules(t *testing.T) {
   }`,
 		},
 		{
-			EnableNamespaces: true,
+			EnableNamespaces: false,
+			EnablePartitions: true,
+			EnablePeering:    false,
+			PartitionName:    "part-1",
+			Expected: `
+partition "part-1" {
+  mesh = "write"
+  acl = "write"
+  node_prefix "" {
+    policy = "write"
+  }
+    policy = "write"
+    acl = "write"
+    service_prefix "" {
+      policy = "write"
+      intentions = "write"
+    }
+    identity_prefix "" {
+      policy = "write"
+      intentions = "write"
+    }
```

Comment:
You can see the bug here where `acl = "write"` winds up being included twice in the root of the policy whenever namespaces are disabled.

The later commit in this PR that adjusts the test assertions after the fix should indicate no scenario in which `acl` or any other type of rule is included twice at the same level.